src/components/Users/UserListAndCard.tsx

```diff
@@ -116,7 +116,9 @@ const UserCard = ({ user }: { user: UserBase }) => {
               </div>
             </div>
           </div>
-          <div className="mt-auto pt-2">{GetDetailsButton(user.username)}</div>
+          <div className="mt-auto pt-2 justify-end">
+            {GetDetailsButton(user.username)}
```

Comment:
although was there previously, let's switch to using proper casings 

https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/typescript-best-practices.html

<img width="790" alt="image" src="https://github.com/user-attachments/assets/681f8cd3-20d2-4db8-8668-aecbfcc6eb9d" />


```suggestion
            {getDetailsButton(user.username)}
```

Reply:
Currently the function uses a hook, #10659 moves it out to the parent component. 🤔  So could copy over the logic or let that PR take care of it.

cc: @rithviknishad